pgns/129041.js

```diff
@@ -61,6 +61,20 @@ module.exports = [
       )
     }
   },
+  {
+    node: 'virtual',
+    value: function (n2k) {
+      const flag  = n2k.fields['Virtual AtoN Flag']
+      return typeof flag != 'undefined' ? flag === 'Yes' : undefined
```

Comment:
`flag === 'Yes' || undefined` would be easier to digest for me, but I'm ok with this.

Reply:
This would not have the same result, right? If flag is No, it should be false, not undefined